02-wrangling.Rmd

```diff
@@ -23,14 +29,23 @@ By the end of the chapter, students will be able to:
     - `group_by`
     - `map`
 
-## Vectors and Data frames
+## Data frames and Vectors
 
-At this point, we know how to load data into R from various file formats. Once loaded into R, most of the tools we have learned about for reading data into R represent the data as a data frame. So now we will spend some time learning more about data frames in R so that we have a better understanding 
-of how we can use and manipulate these objects.
+We now know how to load data into R from various file formats.
+Given that most of the tools we have use to do this represent the data as a 
+data frame in R, we will spend some time learning more about data frames
+to deepen our understanding of them. This will be useful prerequisite knowledge
+to use and manipulate these objects in our data analysis.
 
 ### What is a data frame?
 
-Let's first start by defining what a data frame is exactly. From a data perspective, it is a rectangle where the rows are the observations:
+To define a **data frame**, first, let's explain a few commonly used terms in statistics:
+
+- **observation** -  all of the quantities or a qualities we collect from a given entity/object 
+- **variable** - any characteristic, number, or quantity that can be measured or collected 
+-  **value** - a single collected quantity or a quality from a given entity/object
+
+From a data perspective, a data frame is a rectangle where the rows are the observations:
 
 ```{r 02-setup, include=FALSE}
 devtools::install_github("ttimbers/canlang")
```

Comment:
We need to remove this. We should never install a package on someone else's computer in an Rmd/script. I have included the package in the Docker container.
```suggestion
```